src/compiler/crystal/tools/playground/public/settings.js

```diff
@@ -39,8 +39,8 @@ Playground.settings = {
       console.error("Unable to save settings since localStorage is not available");
     }
   },
-  getGithubToken: function() {
-    return Playground.settings._readSetting('settingsGithubToken');
+  getGitHubToken: function() {
+    return Playground.settings._readSetting('settingsGitHubToken');
```

Comment:
Possibly an api break

Reply:
Yeah, I wouldn't change that to void breaking anyone's settings.
Renaming the method is probably fine. but it might be better to keep the same spelling as the setting name..